.github/workflows/ci.yml

```diff
@@ -118,7 +118,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [3.8, 3.9, '3.10']
+        python-version: [3.8, 3.9, '3.10.6']
```

Comment:
i think we can back this out now

Reply:
ouch-- i rolled back to 3.10 and now see lots of red

Reply:
Yeah I saw something with pandas throwing a type error, didn't have time to dig, but guessing a few other upstream changes to fix against ... I think we can land this tho, as they are unrelated 

Maybe run tests with some error tolerance / skip type checks locally just to ensure not related to your PR then we can land

Chatting w Alex, I'd like to show you the release process, so we can schedule 20min to do together